torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -493,15 +493,15 @@ void ExprSegmentationSorter::resetLevels() {
   std::vector<ExprGroup*> next_to_visit;
 
   while (!to_visit_.empty()) {
-    auto visit = to_visit_.front();
+    auto handle = to_visit_.front();
```

Comment:
Do you really want to rename this variable? Or is it just accidentally done together with the dispatch functions?

Reply:
Accidental, thanks.